app/views/admin/organizations/_location_fields.html.haml

```diff
@@ -1,29 +1,24 @@
-.card.bg-light
-  .card-body
-    .form-group
-      = f.label :name, "Name"
-      = f.text_field :name, class: "form-control"
+%fieldset
+  .card.bg-light
+    .card-body
+      .form-group
+        = f.label :name, "Name"
+        = f.text_field :name, class: "form-control"
 
-    .form-group
-      = f.label :phone, "Phone number"
-      = f.text_field :phone, class: "form-control"
+      .form-group
+        = f.label :phone, "Phone number"
+        = f.text_field :phone, class: "form-control"
 
-    / .form-group
-    /   = f.label :email, "Shop's email (only if you want)"
-    /   .controls
-    /     = f.text_field :email
-
-    .form-group
-      = f.label :country_id, "Address"
-      = f.text_field :street, placeholder: "Street", class: "form-control mt-1"
-    .form-group
-      = f.text_field :city, placeholder: "City", class: "form-control mt-1"
-      = f.collection_select(:state_id, State.all, :id, :name, {prompt: "State", include_blank: true}, class: "form-control mt-1" )
-    .form-group
-      = f.text_field :zipcode, placeholder: "Zipcode", class: "form-control mt-1"
-      = f.collection_select(:country_id, Country.all, :id, :name, {prompt: "Choose country"}, class: "form-control mt-1" )
-    .form-group
-      = f.check_box :_destroy
-      = f.label :_destroy, "Remove", class: "remove_fields"
-
-%hr
+      .form-group
+        = f.label :country_id, "Address"
+        = f.text_field :street, placeholder: "Street", class: "form-control mt-1"
+      .form-group
+        = f.text_field :city, placeholder: "City", class: "form-control mt-1"
+        = f.collection_select(:state_id, State.all, :id, :name, {prompt: "State", include_blank: true}, class: "form-control mt-1" )
+      .form-group
+        = f.text_field :zipcode, placeholder: "Zipcode", class: "form-control mt-1"
+        = f.collection_select(:country_id, Country.all, :id, :name, {prompt: "Choose country"}, class: "form-control mt-1" )
+      .form-group
+        = f.check_box :_destroy
+        = f.label :_destroy, "Remove", class: "remove_fields"
```

Comment:
The checkbox for this used to be hidden, and this was styled. One of your other PRs removed the classes that set styles for it. You should add some styles back!